inc/js_scripts/admin/main.js

```diff
@@ -91,6 +116,11 @@
 		render: function () {
 			this.setTitleColor( this.model.get( 'header_color' ) );
 			this.setContentColor( this.model.get( 'content_color' ) );
+			this.$el.find( '#site-logo' ).attr( 'src', post_modifier.settings.site_logo_src );
+			this.logos.each( model => {
+				this.$el.find( '.actual-preview' ).find( '#' + model.get( 'id' ) ).attr( 'src', model.get( 'src' ) );
```

Comment:
```suggestion
				this.$el.find( '.actual-preview #' + model.get( 'id' ) ).attr( 'src', model.get( 'src' ) );
```

Reply:
you can also use template literals ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals ) : this.$el.find( `.actual-preview #${model.get( 'id' )}` )